src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/hooks/useFeatureEnabled.tsx

```diff
@@ -27,21 +29,25 @@ const useFeatureEnabled = (): UseFeatureEnabledReturn => {
     }
   })
 
-  const [hasPermission] = usePermission(
+  const [canEdit, canToggle] = usePermission(
     {
       resource: {
         resourceType: ResourceType.ENVIRONMENT,
         resourceIdentifier: activeEnvironment
       },
-      permissions: [PermissionIdentifier.TOGGLE_FF_FEATUREFLAG]
+      permissions: [PermissionIdentifier.EDIT_FF_FEATUREFLAG, PermissionIdentifier.TOGGLE_FF_FEATUREFLAG]
     },
     [activeEnvironment]
   )
 
+  const hasPermission = canEdit || canToggle
+
   return {
     enabledByPermission: hasPermission,
     enabledByPlanEnforcement,
-    featureEnabled: hasPermission && enabledByPlanEnforcement
+    featureEnabled: hasPermission && enabledByPlanEnforcement,
+    canEdit,
```

Comment:
(cont from above) then just return:

```suggestion
    canToggle: permission === PermissionIdentifier.TOGGLE_FF_FEATUREFLAG && canToggle,
    canEdit: permission === PermissionIdentifier.EDIT_FF_FEATUREFLAG && canEdit
```